index.html

```diff
@@ -284,26 +284,41 @@ <h3 id="w3c-coordination">W3C Groups</h3>
 
           <dl>
               <dt>
-                  <a href="https://w3c-ccg.github.io/">Credentials Community Group</a>
+                <a href="https://www.w3.org/2017/vc/WG/">Verifiable Credentials Working Group</a>
               </dt>
               <dd>
-                  Coordination on other specifications related to Decentralized Identifiers.
+                Coordination on the use of Decentralized Identifiers in Verifiable Credential ecosystems.
```

Comment:
Why is this restricted to Verifiable Credential ecosystems?
```suggestion
                Coordination on the use of Decentralized Identifiers.
```

Reply:
It's restricted to VC ecosystems because the liaison is the VCWG... they can't officially speak to ecosystems other than VC ecosystems. :)

Reply:
That makes sense. I missed the context that this only applies to the liason relationship.